util/bunzip.go

```diff
@@ -48,3 +49,42 @@ func Bunzip2File(dst, src string) error {
 	}
 	return err
 }
+
+// Bunzip2File does bunzip2 decompression from src file into dst file.
+func Bunzip2File(dst, src string) error {
+	lbunzip2, err := exec.LookPath("lbunzip2")
+	if err != nil {
+		return Bunzip2FileGo(dst, src)
+	}
+
+	cmd := exec.Command(lbunzip2, "--stdout", "--decompress", src)
+	cmd.Stderr = os.Stderr
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		return fmt.Errorf("setup stdout pipe: %w", err)
+	}
+
+	out, err := os.OpenFile(dst, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0644)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		if err != nil {
```

Comment:
What `err` should this refer to?

Reply:
Ok, so this is any of the `err` variables following. I think it's a bit fragile if someone touches this a later line to be `val, err := something` because it will shadow. I think either a comment that one should not use shadowing would be good or move the `os.Remove` call before each `return fmt.Errorf`.

Reply:
Ill add the os.Remove calls